tests/containers/bci_test.pm

```diff
@@ -36,18 +36,18 @@ sub run_tox_cmd {
     $cmd .= " -k \"$bci_marker\"" if $bci_marker;
     $cmd .= " --reruns $bci_reruns --reruns-delay $bci_reruns_delay";
     $cmd .= "| tee $tox_out";
-    record_info("tox", "Running command: $cmd");
+    record_info("tox $env", "Running command: $cmd");
```

Comment:
This gets very soon very long and ugly. Let's please keep the old one in the title.

Reply:
I suggest to keep at least the first word: I will add `(split(' ',$env))[0]`

Reply:
VR
http://mdati-openqa.qe.suse.de/tests/677#step/bci_test_docker/56

Reply:
VR
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18035

Reply:
VR http://mdati-openqa.qe.suse.de/tests/678#

Reply:
VR http://mdati-openqa.qe.suse.de/tests/678#step/bci_test_docker/56